src/libraries/System.Security.Cryptography.X509Certificates/src/Resources/Strings.resx

```diff
@@ -64,7 +65,7 @@
     <value>Array may not be empty or null.</value>
   </data>
   <data name="Arg_EmptyOrNullString" xml:space="preserve">
```

Comment:
Recommend renaming

Reply:
Created new `Arg_Name_EmptyOrNullString` resource as it was used for other exception without passing the parameter